xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java

```diff
@@ -5,13 +5,16 @@
 public class KrakenDepositAddress {
 
   protected final String address;
+  protected final String tag;
   private final Long expiretm;
   private final boolean newAddress;
 
-  public KrakenDepositAddress(@JsonProperty("address") String address, @JsonProperty("expiretm") Long expiretm,
+  public KrakenDepositAddress(@JsonProperty("address") String address,
+      @JsonProperty("tag") String tag, @JsonProperty("expiretm") Long expiretm,
       @JsonProperty("new") boolean newAddress) {
     super();
     this.address = address;
+    this.tag = tag;
```

Comment:
I don't see the tag field in the API docs: https://www.kraken.com/help/api#private-user-funding

Reply:
True. It's not in the doc.  I tested out....

Reply:
Are you saying `tag` is in the response based on your experience?

Reply:
Yes, I tried 2 coins which need tag/memo, and tag is always returned in that case.